src/js/hmm/api.js

```diff
@@ -1,9 +1,46 @@
+/**
+ * Functions for communication with API endpoints related to managing HMMs.
+ *
+ * @module hmm/api
+ */
 import { Request } from "../app/request";
 
-export default {
-    find: ({ term, page }) => Request.get("/hmms").query({ find: term, page }),
-    list: ({ page }) => Request.get("/hmms").query({ page }),
-    install: () => Request.post("/hmms/status/updates"),
-    get: ({ hmmId }) => Request.get(`/hmms/${hmmId}`),
-    purge: () => Request.delete("/hmms"),
-};
+/**
+ * Sagas compatible function for fetching a list of HMMs from the API.
```

Comment:
I think 'sagas compatible' is misleading. These functions return promises. That's it.

Putting this phrase at the beginning of the string also makes the developer scan to the end to see what the function actually does.

```suggestion
 * Fetch a page of HMM search results from the API.
```